app/controllers/katello/api/v2/host_packages_controller.rb

```diff
@@ -102,13 +102,9 @@ def find_host
       @host = resource_finder(::Host::Managed.authorized(:view_hosts, ::Host::Managed), params[:host_id])
     end
 
-    def valid_package_name?(package_name)
-      package_name =~ /^[a-zA-Z0-9\-\.\_\+\,]+$/
-    end
-
     def validate_package_list_format(packages)
       packages.each do |package|
-        if !valid_package_name?(package) && !package.is_a?(Hash)
+        unless package.kind_of?(String) && ::Katello::Util::Package.valid_package_name_format(package).nil?
```

Comment:
This controller change wasn't necessary but I think it's a little more readable - and got to reuse the Util method